v2/unified-login/multiple-frontends-with-separate-backends.mdx

```diff
@@ -303,6 +303,18 @@ class App extends React.Component {
 
 ```
 
+:::important
+If you are using `useRoutes` or `createBrowserRouter` or have routes defined in a different file, use it in the following way. [Issue Reference](https://github.com/supertokens/supertokens-auth-react/issues/581#issuecomment-1246998493)
+```tsx
+const routes = getSuperTokensRoutesForReactRouterDom(reactRouterDom, [^{recipePreBuiltUINameCapitalLetters}]);
```

Comment:
This doesn't work here at all, it's rendered as:
`const routes = getSuperTokensRoutesForReactRouterDom(reactRouterDom, [^{recipePreBuiltUINameCapitalLetters}]);` (without any replacement)